### PR TITLE
Fix selecting the shearched objects that are provided using console

### PR DESCRIPTION
**Pull request by vahedi25411**
This should fix the error in specifying `1` for `periodic_counter_time` and also selecting the objects that we want to have in our export file using CLI.

### CODE REVIEW

File: automated_walk_bike_counter/core/configuration.py [hunk 1]
```diff
@@ -195,9 +199,9 @@ def list_of_tuples(arg):
 
 parser.add_argument(
     "--search_objects",
-    type=list,
+    type=list_of_items,
     default=["Person", "Cyclist"],
-    help="The list of valid objects.",
+    help="The list of valid objects comma separated like Person,Cyclist.",
```
Comment (ian-r-rose) [inline automated_walk_bike_counter/core/configuration.py:204]: Oh, interesting! I am not sure what the difference between this option and the `VALID_OBJECTS` option is. Can you help me understand @vahedi25411 ?

File: automated_walk_bike_counter/core/tracking/object_tracker.py [hunk 2]
```diff
@@ -408,9 +408,11 @@ def trackObjects(self, args):
             # input counter time in minutes * video frame per second * number of
             # seconds in each min
             self.periodic_counter_interval = int(
-                # config.periodic_counter_time * self.video.fps * 60
                 config.periodic_counter_time
                 * self.video.fps
+                * 60
+                # config.periodic_counter_time
```
Comment (ian-r-rose) [inline automated_walk_bike_counter/core/tracking/object_tracker.py:414]: Can we just remove this commented-out-code if it's not doing anything?

Comment (vahedi25411) [reply automated_walk_bike_counter/core/tracking/object_tracker.py]: Sure.

File: automated_walk_bike_counter/core/configuration.py (hunk 1 above)
Comment (ian-r-rose) [reply automated_walk_bike_counter/core/configuration.py:204]: No, that makes sense to me, I just misunderstood. Thanks!

**Comment by ian-r-rose** [comment]
Interesting, the fix here looks good to me, but CI is showing a weird difference in the formatting of the time string: https://dev.azure.com/ianrose/automated-walk-bike-counter/_build/results?buildId=291&view=logs&j=4b6540fc-ceaa-5116-41a8-c00db5b232e4&t=b99515d0-4181-5b81-7a22-773a999e6d76&l=3132 .

Do you have any idea what might be going on there @vahedi25411? I'm puzzled by it. Perhaps some weird localization of the time output?

**Comment by vahedi25411** [comment]
It is a weird result, and I don't have any idea what the problem is. It shouldn't be for localization because CI doesn't have any sense about data type here. Maybe excel detected this column as time, while it should be considered as text.

**Comment by ian-r-rose** [comment]
Okay, I've made two changes:
1. Used the standard `timedelta` for outputting the time, which is hopefully more reliable than the hand-rolled solution.
1. Tried to normalize CSV output to use `\n` line endings. Hopefully this will fix some of the differences in output between Windows and POSIX.

@vahedi25411 does this look reasonable to you?

**Comment by vahedi25411** [comment]
Yes, it makes sense. Did these changes fix the issue?

**Comment by ian-r-rose** [comment]
Yep, issue is fixed, CI is happy. Thanks @vahedi25411 !

**Pull request merged**